build-tasks/templates/rule.template.js

```diff
@@ -3,9 +3,7 @@ module.exports = ({ ruleName, walkerName }) =>
 import * as Lint from 'tslint';
 
 import {ExtendedMetadata} from './utils/ExtendedMetadata';
-// use (and contribute to) AstUtils for common AST functions // TODO: delete comment
 import {AstUtils} from './utils/AstUtils';
-// use Utils instead of Underscore functions // TODO: delete comment
```

Comment:
Looks like it is comment for developer who is creating new rule. Not sure if they should be removed

Reply:
Ah okay, I thought so. My bad.